rocAL/rocAL/include/augmentations/effects_augmentations/node_jitter.h

```diff
@@ -25,18 +25,17 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-
-class JitterNode : public Node
-{
+class JitterNode : public Node {
```

Comment:
Instead of inheriting as public, can we inherit them as private in the augmentation nodes?
Will that lead to any issues?

Reply:
Yes we don't want to do a private inheritance and this is the norm we follow in all files